### PR TITLE
[operations #2] added operations for assets and remove content op models

### PR DESCRIPTION
**Pull request by vanderian**

### CODE REVIEW

File: library/build.gradle.kts [hunk 1]
```diff
@@ -11,6 +10,7 @@ plugins {
 
 dependencies {
   implementation(Libs.kotlin)
+  implementation(Libs.kotlin_reflect)
```
Comment (lukas1) [inline library/build.gradle.kts]: why is this necessary? Because of `@UIntXX` annotations?

Comment (vanderian) [reply library/build.gradle.kts]: no, not sure, there was a warning for different version
will check dependency tree where this requirement comes from
